lib/python/qmk/commands.py

```diff
@@ -61,3 +65,20 @@ def parse_configurator_json(configurator_file):
     user_keymap = json.load(configurator_file)
 
     return user_keymap
+
+
+def run(command, *args, **kwargs):
+    """Run a command with subprocess.run
+    """
+    platform_id = platform.platform().lower()
+
+    if isinstance(command, str):
+        raise TypeError('`command` must be a non-text sequence such as list or tuple.')
+
+    if 'windows' in platform_id:
+        safecmd = map(shlex.quote, command)
+        safecmd = ' '.join(command)
+        shell = os.environ.get('SHELL', '/bin/sh')
```

Comment:
`SHELL` must be a Windows path, and we apparently can't know the MSYS base directory to tack on `/bin/sh`, so having a default here won't work.